migu.go

```diff
@@ -710,12 +736,21 @@ func makeStructAST(d dialect.Dialect, name string, schemas []dialect.ColumnSchem
 	}, nil
 }
 
-func parseStructTag(d dialect.Dialect, f *field, tag reflect.StructTag) error {
+func hasEnumColumn(schemas []dialect.ColumnSchema) bool {
+	for _, schema := range schemas {
+		if schema.IsEnumerated() {
+			return true
+		}
+	}
+	return false
+}
+
+func parseStructTag(d dialect.Dialect, f *field, tag reflect.StructTag, separator string) error {
 	migu := tag.Get("migu")
 	if migu == "" {
 		return nil
 	}
-	for _, opt := range strings.Split(migu, ",") {
+	for _, opt := range strings.Split(migu, separator) {
```

Comment:
@tsuty I would like to keep the current syntax yet.

```go
migu:"type:enum('A','B','O','AB'),null"
```

Therefore, I think it is better to implement [bufio.Scanner](https://golang.org/pkg/bufio/#Scanner) to parse options correctly, then use it rather than `separator` annotation.